Update ISO.csv

sim55649 opened the pull request:
I've prepared this pull request where I report the following updates on the values taken from the https://www.worldbank.org dataset:
- Updating of population data by taking them from the dataset (greater coverage of data consistent with the subsequent indicators inserted)
- Indicator relating to% of the female gender out of the total population
- Indicator relating to% of female smokers
- Indicator relating to% of male smokers

Since some of these data refer to the year 2016, I have left this aspect indicated in the column.

If it should be all right I can move on to editing the _src file.

sim55649 committed:
Update ISO.csv

 I've prepared this pull request where I report the following updates on the values taken from the https://www.worldbank.org dataset:
- Updating of population data by taking them from the dataset (greater coverage of data consistent with the subsequent indicators inserted)
- Indicator relating to% of the female gender out of the total population
- Indicator relating to% of female smokers
- Indicator relating to% of male smokers

Since some of these data refer to the year 2016, I have left this aspect indicated in the column.

If it should be all right I can move on to editing the _src file.

sim55649 committed:
Update ISO.csv

sim55649 committed:
Update ISO.csv

fix quotation marks

sim55649 committed:
Update ISO.csv indicators

- Added indicators from:
https://data.worldbank.org/indicator/NY.GDP.MKTP.CD
https://data.worldbank.org/indicator/NY.GDP.PCAP.CD
https://data.worldbank.org/indicator/SH.MED.BEDS.ZS
https://data.worldbank.org/indicator/SH.MED.CMHW.P3

Commit:
clean ISO and add src

eguidotti commented:
Thanks! I have cleaned the column names and noticed that country names containing commas were changed. Fixed.

Please add to the `_src.csv` the source of the data (at the very bottom of the file you can find examples). For data with mixed years you can write there eg 2016-2018 (column `year`).

sim55649 committed:
Update with new indicators

- Updated Smoking Female/Male data
- Added 2 new indicators from WorldBank:
1. https://data.worldbank.org/indicator/SH.XPD.CHEX.GD.ZS
2. https://data.worldbank.org/indicator/SH.XPD.OOPC.CH.ZS

sim55649 committed:
Update _src.csv

added references to the latest indicators

Commit:
renaming columns

eguidotti commented:
Excellent work! Thank you!
